source/security/index.html.md.erb

```diff
@@ -74,6 +76,16 @@ take care not to use any real card numbers. Read more about [testing](/testing_g
 
 GOV.UK Pay does not store full card numbers or CVV data for security reasons. This means you will not be able to search for transactions using card numbers. You can, however, search for the first six digits and the last four digits of a card.
 
+## Preventing fraudulent payments
+
+If you think you’re receiving fraudulent payments, you can:
```

Comment:
looking at Katie's notes, we might want to emphasise that this is mostly handled with your PSP but that we have some tools to help. Or do we think it's clear enough?